src/components/Highlighter/GroupHighlighter/GroupHighlighter.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { HighlighterCore, HighlighterCoreAccuracy } from '../../../packages/@highlighter/core';
+import { HighlighterControl } from '../../../packages/@highlighter/control';
+import { HighlighterNode, HighlighterNodeRenderer } from '../../../packages/@highlighter/node';
+
+type GroupHighlighterTermResult = {
+    styles: CSSStyleDeclaration;
+};
+interface Props {
+    /** Target DOM node for highlight searching */
+    children: React.ReactElement;
+    /** Array of words or RegExp that is used for search */
+    searchTerms: string[];
+    /** Method is called on render founded substrings */
+    highlightRenderer: HighlighterNodeRenderer;
+    /**
+     * Define the accuracy of search
+     * Supported options:
+     * partial - when searching for "lor" only "lor" inside "lorem" will be marked
+     * exact - When searching for "lor" only those exact words with a word boundary will be marked.
+     * regexp - When searching for "lor" only those exact words with a word boundary will be marked.
+     */
+    accuracy?: HighlighterCoreAccuracy;
+    /** If enabled the diacritic characters should be matched */
+    ignoreDiacritics?: boolean;
+    /** Ignore case sensitivity */
+    ignoreCase?: boolean;
+    /** Fired after finishing highlighting */
+    onComplete: (
+        instance: HighlighterControl,
+        result: DictionaryOf<GroupHighlighterTermResult>
+    ) => void;
+}
+
+/**
+ * Defines styles for highlighted node
+ */
+const HIGHLIGHTED_NODE_STYLES: CSSStyleDeclaration = {
+    borderRadius: '2px',
+} as CSSStyleDeclaration;
+
+/**
+ * Defines styles for selected node
+ */
+const SELECTED_NODE_STYLES = {
+    boxShadow: '0 0 0 1px #000',
+    position: 'relative',
+} as CSSStyleDeclaration;
+
+/**
+ * Defines styles for cleaning selection for node
+ */
+const RESET_SELECTED_NODE_STYLES: CSSStyleDeclaration = {
```

Comment:
is it necessary to duplicate `CSSStyleDeclaration` here?

Reply:
I forgot to delete it, thanks